server/liveblog/themes/template/filters.py

```diff
@@ -71,7 +70,20 @@ def addten(date):
 
 
 def ampify(html):
-    if re.search('iframe', html, re.IGNORECASE):
+    brightcove_re_array = [
```

Comment:
the string is to big to pass `flake8` and spiting into the next line doesn't look so good
so list that is joined into a string looks much better IMHO